wallet/src/actors/worker/methods.rs

```diff
@@ -1062,9 +1062,16 @@ impl Worker {
         sink: DynamicSink,
     ) -> Result<bool> {
         // Only trigger sync of chain if data has been cleared
```

Comment:
```suggestion
        // Only trigger sync of chain if the wallet is not already resyncing
```
The function documentation is also outdated. What would happen if the resync is started while a normal sync is already in place?